v3/lints/cabf_br/lint_subject_contains_organizational_unit_name.go

```diff
@@ -0,0 +1,64 @@
+package cabf_br
```

Comment:
@mtgag I believe that #675  captures this requirement effectively, so perhaps we can simply delete this file and reduce this change to `SubjectContainsOrganizationalUnitNameButNoOrganizationName`.

@sleevi you were the primary blocker on this PR, were your concerns addressed (looking to move forward on this one and #675)?

Reply:
Right, I think that leaving the date update in this PR is positive since it was a good bugfix to find that the other PR does not capture.

Reply:
Okay, thank you for your patience @mtgag I went ahead and merged #675 so I believe that this particular lint can be removed and we can move forward with the other changes in this PR.